ch_backup/backup/metadata/table_metadata.py

```diff
@@ -53,6 +55,24 @@ def get_parts(self, *, excluded_parts: Set[str] = None) -> List[PartMetadata]:
                     PartMetadata.load(self.database, self.name, part_name, raw_metadata)
                 )
 
+        if self.engine == REPLACING_MERGE_TREE:
+
+            def split_part_name(part):
+                chunks = part.split("_", maxsplit=3)
+                partition = ""
+                suffix = ""
+                try:
+                    partition = chunks[0]
+                    start_range = int(chunks[1])
+                    finish_range = int(chunks[2])
+                    suffix = chunks[3]
+                except (IndexError, ValueError):
+                    start_range = 0
+                    finish_range = 0
+                return partition, start_range, finish_range, suffix
+
+            result.sort(key=lambda part: split_part_name(part.name))
```

Comment:
Please, update the docstring of the function that it returns list in the sorted order. 
BTW, wherever this function is called, do we need a sorted order?
Maybe enable that by parameter or make as a separate function.

Reply:
I think we dont need get_parts without sorting. I suggest using sorted parts as result this function in all cases.